framework/packages/abstract-sdk/src/ans_resolve.rs

```diff
@@ -86,24 +73,11 @@ impl Resolve for AnsAsset {
     }
 }
 
-impl Resolve for &AnsAsset {
```

Comment:
Why was this removed?

Reply:
> Why was this removed?

It was not used and it's weird to implement trait for &&T